plugins/domain-mapping.php

```diff
@@ -3,112 +3,109 @@
 function domain_mapping_gc_cache( $function, $directory ) {
 	global $cache_path;
 
-	if ( ! function_exists( 'domain_mapping_warning' ) ) {
-		return false;
+	$siteurl = function_exists( 'domain_mapping_warning' ) ? domain_mapping_siteurl( false ) : false;
+	if ( false === $siteurl ) {
+		return;
```

Comment:
I'm not a huge fan of this kind of optimization. It makes it harder to see what's going on. Can you change this back to how it was originally?

Reply:
Yes 👍 . I replaced `return false;` with `return;` because function doesn't return value.